python/sbp/acquisition.py

```diff
@@ -20,7 +20,7 @@
 
 from sbp.msg import SBP, SENDER_ID
 from sbp.utils import fmt_repr, exclude_fields, walk_json_dict, containerize
-from sbp.gnss import *
+from sbp.gnss import GnssSignal, GnssSignalDep, GPSTime, CarrierPhase, GPSTime, GPSTimeSec, GPSTimeDep, SvId  # noqa
```

Comment:
I find this pylint rule fascinating, you could use 90% of a file and it still prefers you write out all imports as long as one is not imported.

Reply:
Hmm yeah that could definitely be problematic. Thanks for sharing.

Reply:
fwiw, I was the one who added this rule to pyflakes: https://github.com/PyCQA/pyflakes/pull/46 .  And the errors it generates drives the fixer [`autoflake --expand-star-imports`](https://pypi.org/project/autoflake/)